docs/compartmentalizing_dav1d.md

```diff
@@ -0,0 +1,577 @@
+# Compartmentalizing `dav1d`
+
+This is a thorough overview of the steps taken to compartmentalize the AV1 decoder `dav1d`.
+
+The necessary changes can be found in [`immunant/dav1d/ia2`](https://github.com/immunant/dav1d/tree/ia2),
+which was branched off from [`videolan/dav1d/master`](https://code.videolan.org/videolan/dav1d)
+to [`immunant/dav1d/master`](https://github.com/immunant/dav1d/tree/master),
+and the changes can be seen in [`master..ia2`](https://github.com/immunant/dav1d/compare/master...ia2).
+
+## Building IA2
+
+The IA2 tools that we need to compartmentalize and run `dav1d` are:
+
+| Binary Name             | `ninja` Target Name       |
+| ----------------------- | ------------------------- |
+| `ia2-rewriter`          | `rewriter`                |
+| `pad-tls`               | `pad-tls`                 |
+| `libpartition-alloc.so` | `partition-alloc-padding` |
```

Comment:
The target should've been `partition-alloc` so we should add that when updating the docs. Both build the same thing, but the one you used is an artifact of cmake limitations and only intended to be used internally.

Reply:
Ahh, I didn't realize.  Let me fix that.

Reply:
Fixed in 6cf6549.